src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -188,6 +188,7 @@ public void testWriteTagRecordsAndCompactCtors() throws Exception {
             getNonCompilablePath("InputWriteTagRecordsAndCompactCtors.java"), expected);
     }
 
+
     @Override
     protected void verify(Checker checker,
                           File[] processedFiles,
```

Comment:
this test is giving an error after my change. it uses a strange verify method overridden from the abstractTestSupport that doesn't verify the actual violation with the expected array and just reads the line (violation comment ) and asserts that it is equal to the expected array but the check doesn't getTheActualViolation by running check on the file like we do normally. may be I am missing something so please correct me because this tests are falling after my optimization